src/packages/range/range.taro.tsx

```
@@ -288,70 +278,64 @@ export const Range: FunctionComponent<
-        total = rect.height
-        diff = (delta / total) * scope
-      }
 
-      let newValue
-      if (isRange(startValue)) {
-        newValue = (exactValue as number[]).slice()
-        newValue[buttonIndex] = startValue[buttonIndex] + diff
-      } else {
-        newValue = startValue + diff
+      const handleMove = async () => {
+        const rect = await getRectInMultiPlatform(root.current)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)